src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -147,8 +148,8 @@ const TeamMemberTasks = props => {
                     </td>
                     <td className="team-clocks">
                       <u>{user.weeklyComittedHours ? user.weeklyComittedHours : 0}</u> /
-                      <font color="green"> {Math.round(totalHoursLogged)}</font> /
-                      <font color="red"> {Math.round(totalHoursRemaining)}</font>
+                      <font color="green"> {thisWeekHours?thisWeekHours.toFixed(1):0}</font> /
+                      <font color="red"> {totalHoursRemaining?totalHoursRemaining.toFixed(1):0}</font>
```

Comment:
"Remaining hours" column has been changed though not mentioned on description. This change adds decimals in the data. Refer screenshot. Please confirm if this is necessary.

Reply:
I agree. That makes sense.

Reply:
Agreeing that this PR is focusing on label change and limiting middle column data to current week's effort.